tests/expose_test.go

```diff
@@ -633,19 +626,19 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			})
 
 			AfterEach(func() {
-				Expect(cleanupService(serviceName, vm.GetNamespace())).To(Succeed(), "should be able to delete the k8sv1.Service entity")
+				Expect(cleanupService(serviceName, tests.NamespaceTestDefault)).To(Succeed(), "should be able to delete the k8sv1.Service entity")
```

Comment:
Because one of the VM tests (the last one) deletes the vm. 

As such, I don't have a VM to get the name from.

I can add a variable to store the name instead.

Reply:
I think this `AfterEach` is redundant anyway since as I mentioned in other places, I would move `tests.BeforeTestCleanup()` to `beforeEach` and remove the `AfterEach`es.

Reply:
Yes, the comment was about the vm related tests only.

Reply:
Removed all the `AfterEach` clauses on the VM related tests.

"Resolving" this.